declarations/Flutterwave.json

```
@@ -0,0 +1,9 @@
+{
+    "name": "Flutter wave",
```

I see in the terms that the name of the service is spelled “Flutterwave”, with no space 🙂 

On top, the service ID (name of the file) is properly spelled without a space, so this service name should reflect it.

```suggestion
    "name": "Flutterwave",
```

---

Thanks for noticing that. I made a mistake there despite getting it right in the pull request title. How clumsy😂

---

It's an easy mistake! 🙂 And very useful to see how easy it is to make, this will make you more attentive to it when reviewing 😊